Gradata/src/gradata/_doctor.py

```
@@ -196,12 +226,214 @@ def _check_disk_space(brain_path):
+        method="GET",
+    )
+    try:
+        with urllib.request.urlopen(req, timeout=_CLOUD_PROBE_TIMEOUT) as resp:
+            body = resp.read(512).decode("utf-8", errors="replace")
+            return resp.status, body
+    except urllib.error.HTTPError as e:
+        body = ""
+        try:
+            body = e.read(512).decode("utf-8", errors="replace")
+        except Exception:
+            pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)